zproject/prod_settings_template.py

```diff
@@ -86,6 +86,10 @@
 # Enable at least one of the following authentication backends.
 # See http://zulip.readthedocs.io/en/latest/prod-authentication-methods.html
 # for documentation on our authentication backends.
+#
+# The install process requires EmailAuthBackend (the default) to be
+# enabled.  If you want to disable it, do so after creating the
```

Comment:
+ "and other backends disabled"?

Reply:
Good question. :) If I'm understanding our old docs and Tim's comment above right, I think other backends being enabled should be harmless (which is why I wrote it this way) -- but would be glad for confirmation from @timabbott .

Reply:
Yep, that's correct.  Though maybe clearer would be "requires using email authentication via EmailAuthBackend..."